evaluation.md

```diff
@@ -2,18 +2,47 @@
 
 ## Internal DSL
 
-_Describe each change from your ideal syntax to the syntax you implemented, and
-describe_ why _you made the change._
+**Describe each change from your ideal syntax to the syntax you implemented, and
+describe_ why _you made the change.**
+
+First, we changed "def" to "state", because def is a reserved keyword in scala, so we couldn't use it.
+
+Next, we got rid of colons after the state name. We tried to make these into a function, but we decided to just use the apply function.
+
+We also had to change the "while" and "if" keywords to "loop" and "cond", since these were also reserved.
+
+We then realized that scala need a TON of parentheses in order to figure out what was going on.... so our code looks like lisp.
+
+Next, we wrapped the individual states into instances of an `OurState` class so that they could be processed properly in our syntax. We did the same thing for all of the states in an `OurProgram` wrapper.
```

Comment:
Was there a reason you couldn't use the State class provided (which _is_ a case class)? (this is a genuine question and not really intended as criticism)